test/integration/narrow/routes/start-application.test.js

```diff
@@ -22,6 +24,31 @@ describe('start-application route', () => {
     address: 'address1, address2, address3, postalCode'
   }]
 
+  const applications = {
+    agreements: [
+      {
+        agreementNumber: 'SFIA123456',
+        agreementData: {
+          land: 'land',
+          funding: 'funding',
```

Comment:
Interesting approach to mocking values that you do not directly care about in the tests -- be interesting to hear what other make of this and whether a mock_cache or mock_models for some entities should exist and tests read it in?

if you remove these unrequired keys, I assume the programme breaks as somewhere a piece of code is accessing them but you are not querying their output, hence this functions correctly?

Reply:
Yeah exactly, the values of those keys are not used during the test so they are just added as mocks to ensure the format of the object is correct. Can change it to a real value if needed.